test/functional/testlib.bash

```diff
@@ -2030,6 +2084,8 @@ add_to_pack() {
 		sudo tar -C "$item_path" -rf "$version_path"/pack-"$bundle"-from-"$version".tar --transform "s,^,staged/," "$(basename "$item")"
 	elif [[ "$item" = *"/delta"* ]]; then
 		sudo tar -C "$version_path" -rf "$version_path"/pack-"$bundle"-from-"$version".tar delta/"$(basename "$item")"
+	elif [[ "$item" = *"/Manifest."*".D"* ]]; then
+		sudo tar -C "$item_path" -rf "$item_path"/pack-"$bundle"-from-"$version".tar "$(basename "$item")"
```

Comment:
There was a discussion on the [mixer delta manifest pull request ](https://github.com/clearlinux/mixer-tools/pull/414/commits/fe5f1788dcdc1fb6177d77b3cf675c65a09cc872) about where to store the manifest inside the pack. It looks like the decision was to store the delta manifest inside the TO version's directory instead of at the top level.


- delta/
    - delta files
- staged/
    - full files
- 25720/
    - Manifest.os-core.D.24710

Reply:
ooh, thanks for pointing out that info @jwakre , I'll update that.